app/gateway-oss/2.5.x/performance-testing-framework.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Performance Testing Framework
+---
+
+## Introduction
+
+Kong codebase includes a performance testing framework (hereinafter called the “framework").
+It allows Kong developers and users to evaluate performance of Kong itself,
+bundled or custom plugins, as well as plot framegraphs to debug performance bottleneck.
+The framework collects RPS (request per second) and latencies of Kong processing the request
+to represent performance metrics under different workloads.
+
+The framework is implemented as an extension to Kong's intergration test suite.
+
+## Installation
+
+The frameworks uses `busted` and some Lua development dependencies of Kong. To setup the environment,
```

Comment:
Add a link to the busted testing framework when referencing 'busted' here. So a user to learn more about the framework.

Reply:
Added!